recipe/build.sh

```diff
@@ -2,18 +2,17 @@
 
 set -ex
 
-if [ $(uname) = Darwin ] ; then
-    LDFLAGS="$LDFLAGS -Wl,-rpath,$PREFIX/lib"
+if [[ "$target_platform" == osx-* ]] ; then
     # dead_strip_dylibs breaks some tests
     LDFLAGS=${LDFLAGS//-Wl,-dead_strip_dylibs/}
-else
-    LDFLAGS="$LDFLAGS -Wl,-rpath-link,$PREFIX/lib"
 fi
 
+export PKG_CONFIG=$BUILD_PREFIX/bin/pkg-config
```

Comment:
@isuruf Do you know why you did this? I would have expected that `PKG_CONFIG` needs to point to `$PREFIX`.